packages/scss/src/components/_textarea.scss

```diff
@@ -59,15 +59,15 @@
 		background-repeat: repeat-x;
 		background-size: 3px 1px;
 		box-shadow: none;
-		color: $greyLighter;
+		color: _get($luTheme, "palette.colors.greyLight");
 		cursor: not-allowed;
 	}
 }
 
 .textarea {
 	&.is-filled {
 		.textarea-label {
```

Comment:
Why do you have to have distinct classes for an input and a textarea label? Is this the case for selects as well?